src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4616,7 +4616,6 @@ internal static SessionStateAliasEntry[] BuiltInAliases
                     new SessionStateAliasEntry("dir", "Get-ChildItem", string.Empty, AllScope),
                     new SessionStateAliasEntry("echo", "Write-Output", string.Empty, AllScope),
                     new SessionStateAliasEntry("fc", "Format-Custom", string.Empty, ReadOnly),
-                    new SessionStateAliasEntry("kill", "Stop-Process"),
```

Comment:
This should maybe be moved up to the `!UNIX` part instead

Reply:
Fixed.